specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/examples/DeletedManagedHsm_Purge.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "name": "hsm1",
+    "location": "westus",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2021-04-01-preview"
+  },
+  "responses": {
+    "200": {},
+    "202": {},
```

Comment:
1. As mentioned elsewhere, this shouldn't return both 200 and 202.
2. That said, what would a 202 even mean here anyway? After a deleted vault / HSM is purged, there's no resources. Key Vault's purge - at least in the data plane (so please check the mgmt plane) only returns 200. The operation is an LRO, but there's nothing to poll.